scripts/common_test.py

```diff
@@ -372,10 +370,8 @@ def mock_isdir(unused_dirpath):
             return False
         def mock_chdir(unused_dirpath):
             pass
-        # pylint: disable=unused-argument
-        def mock_popen(unused_cmd, stdin, stdout, stderr):
+        def mock_popen(unused_cmd, stdin, stdout, stderr):  # pylint: disable=unused-argument
```

Comment:
Not, possible as these are keyword arguments.

Reply:
So why this doesn't look like a keyword argument here? i.e, why not stdin=<something>? Can we do that?

Reply:
+1 to @DubeySandeep's comment, here and elsewhere. 

That said, this can be deferred to a separate PR. Feel free to create an issue for it if you like and link it here. But make sure to have a very clear description of the issue.

Reply:
Ah, I see. Thanks for the explanation; I agree that a solution is not obvious. Let's leave it for now.